File: app/src/main/java/com/example/mobile_challenge/model/QuoteLiveResponse.kt
package com.example.mobile_challenge.model

import kotlinx.serialization.Serializable

@Serializable
class QuoteLiveResponse(
  val success: Boolean,
  val terms: String,
  val privacy: String,
  val timestamp: Long,
  val source: String,
  val quotes: Map<String, Double>
)

File: mobile-challenge/Model/QuoteLiveResponse.swift
//
//  QuoteLiveResponse.swift
//  mobile-challenge
//
//  Created by Kivia on 5/2/20.
//  Copyright © 2020 AP Club. All rights reserved.
//

import Foundation

struct QuoteResponse : Codable {
  
  let success: Bool
  let terms: String
  let privacy: String
  let timestamp: Int64
  let source: String
  let quotes: Dictionary<String, Double>
  
}
